### PR TITLE
Review the project

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
I applied best practices in HTML code.
I used semantic HTML tags, CSS selectors correctly and CSS box model.
Performed a code review for myself on a past project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          </div>
+      
+          <div class="form-group">
+            <button type="submit" id="submit" class="submit-button">
+              Submit
+            </button>
+          </div>
+        </form>
+      </div>
+
```
Comment (ZuhRa-HashiMi) [inline index.html:223]: please kindly delete the extra space for cleaning the code.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+button {
+  border: none;
+}
+
+.container {
+  width: 100%;
+  margin: 3.125rem auto 0 auto;
+}
+
```
Comment (ZuhRa-HashiMi) [inline style.css:85]: please use max-width more than 540px

File: style.css [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+/* form */
+
+form {
+  background: var(--color-darkblue-alpha);
+  padding: 2.5rem 0.625rem;
+  border-radius: 0.25rem;
+}
+
```
Comment (ZuhRa-HashiMi) [inline style.css:126]: please remove the padding, it's not suitable.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+              name="comment"
+              placeholder="Enter your comment here..."
+            ></textarea>
+          </div>
+      
+          <div class="form-group">
+            <button type="submit" id="submit" class="submit-button">
+              Submit
+            </button>
+          </div>
```
Comment (ZuhRa-HashiMi) [inline index.html:220]: Kindly delete all the extra space.
you have to delete because  that you have to make your project clean.

**Review by ZuhRa-HashiMi — COMMENTED** [review]
You have to make  clean code so for this, clean the extra comment and  as well extra space.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+        </form>
+      </div>
```
Comment (ZuhRa-HashiMi) [inline index.html:222]: Please remove the extra line.

File: style.css (hunk 2 above)
Comment (ZuhRa-HashiMi) [inline style.css:85]: you have to add more high for media query.

**Review by ZuhRa-HashiMi — COMMENTED** [review]
To make a clean project so you need to clean the extra line and as well you have to clean extra comment and as well as use semantic tag, so I saw that you already used. but few things reminded.

File: index.html (hunk 5 above)
Comment (ZuhRa-HashiMi) [inline index.html:222]: kindly delete the extra line

File: style.css [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
```
Comment (ZuhRa-HashiMi) [inline style.css:14]: please add some more space to your website's body .

**Review by ZuhRa-HashiMi — COMMENTED** [review]
you have to make clean code.

**Review by Kamba56 — APPROVED** [review]
Hi @ZuhRa-HashiMi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] Polite comments👏👏👏


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [Optional] Please always include an explanation, a "why", for each requested change. For example:

This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*

This request lacks a "why": *Why don't you use an article tag here instead of a div?*

- [Optional] Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**